front/lib/models/assistant/actions/process.ts

```diff
@@ -97,6 +97,20 @@ AgentProcessConfiguration.init(
   },
   {
     modelName: "agent_process_configuration",
+    indexes: [
+      {
+        unique: true,
+        fields: ["sId"],
+        concurrently: true,
+      },
+      {
+        fields: ["agentConfigurationId"],
+        concurrently: true,
+      },
+      {
```

Comment:
Is there a reason to not add the concurrently here?

Reply:
Are we not going to query for `forceUseAtIteration` in conjunction with `agentConfigurationId`?

Reply:
> Are we not going to query for forceUseAtIteration in conjunction with agentConfigurationId?

No, probably not. The idea is to be able to count in aggregate

Reply:
And you are right, I missed  concurrently here 👍  adding thanks